save-backend/src/main/kotlin/com/saveourtool/save/backend/service/OrganizationService.kt

```diff
@@ -69,7 +69,18 @@ class OrganizationService(
      * @param organization
      * @return organization
      */
-    fun updateOrganization(organization: Organization): Organization = organizationRepository.save(organization)
+    @Suppress("UnsafeCallOnNullableType")
+    fun updateOrganization(organization: Organization): Organization = organizationRepository.findByName(
+        organization.name
+    )!!.let { organizationFromDb ->
+        organizationRepository.save(organizationFromDb.copy(canCreateContests = organization.canCreateContests).apply { id = organizationFromDb.id })
+    }
+
+    /**
+     * @param organization
+     * @return organization
+     */
+    fun changeCanCreateContests(organization: Organization): Organization = organizationRepository.save(organization)
```

Comment:
why we need a separate method for it?

Reply:
Because in one method we provide changing `canCreateContests` only and in another we provide changing other parameters. To my mind we should forbid changing canCreateContests flag separately